packages/core/src/connectivity/scp-cf/destination/destination-from-env.ts

```diff
@@ -161,3 +167,26 @@ export function searchEnvVariablesForDestination(
 
   logger.info('No environment variable set.');
 }
+
+function destinationAuthToken(
+  token?: string
+): [DestinationAuthToken] | undefined {
+  if (token) {
+    const decoded = decodeJwt(token);
+    logger.info(
+      "Option 'forwardAuthToken' enabled on destination. Using the initial token for the destination."
+    );
+    return [
+      {
+        value: token,
+        expiresIn: decoded.exp!.toString(),
```

Comment:
If the service token expires, will you get a refreshed token automatically?

Reply:
no, The token we are talking about is the initial one from the incoming request. I would assume that his token is still valid.